requirements/edx/base.txt

```diff
@@ -55,8 +54,9 @@ Markdown==2.2.1
 --allow-unverified meliae
 meliae==0.4.0
 mongoengine==0.7.10
+MySQL-python==1.2.5
 networkx==1.7
-nltk==2.0.4
+nltk>=2.0.4 # upgrade to 2.0.5 once edx-ora2 is compatible
```

Comment:
edx-ora2 requires nltk==2.5. I thought it would be easier to upgrade to nltk==2.0.5 by first upgrading the requirements of edx-ora2, but if we pin the dependency it will not be possible to upgrade without conflict...